api/src/modules/widgets/custom-widgets.controller.ts

```diff
@@ -80,15 +87,16 @@ export class CustomWidgetsController {
     });
   }
 
+  @CheckAuthorization([AUTHORIZATION_STRATEGIES.USER_ID])
   @TsRestHandler(c.deleteCustomWidget)
   public async deleteCustomWidget(
     @GetUser() user: User,
   ): Promise<ControllerResponse> {
     return tsRestHandler(c.deleteCustomWidget, async ({ params }) => {
       const { userId, id } = params;
-      const data = await this.customWidgetsService.removeCustomWidget(
+      const data = await this.customWidgetsService.deleteCustomWidget(
```

Comment:
This is a redundancy in a code that will be executed in lots of request. We are validating the user twice here